content/applications/marketing/surveys/overview/create.rst

```diff
@@ -1,88 +1,278 @@
+employees), learn what the market is *really* thinking, and so much more.
+
+Getting started
+===============
+
+To begin, open the :guilabel:`Surveys` application and click :guilabel:`Create` on the
+:guilabel:`Surveys` dashboard.
+
+Odoo redirects the page to a blank survey template form.
+
+First, choose a :guilabel:`Survey Title` for the survey. Next, add a cover image to the survey by
+hovering over the photo icon and clicking on the pencil icon. Then, the file explorer will open and
+the user can choose an image from their files. However, choosing an image is not required.
 
-When creating your survey, choose a *Title* and *Category*. The *Category* field is used to know in
-which context the survey is being conducted, as different applications might use it for different
-purposes such as recruitment, certification, or employee appraisal.
+Below the :guilabel:`Survey Title` are various tabs in which the survey questions and format can be
+created, customized, and configured. These tabs are labeled as follows: :guilabel:`Questions`,
+:guilabel:`Description`, and :guilabel:`Options`.
 
-.. image:: create/new_survey.png
+.. image:: create/questions-description-options.png
    :align: center
-   :alt: Form view of a survey with a title and category being chosen in Odoo Surveys
-
-Tab: Questions
---------------
-
-| Add sections and questions by clicking on the respective links.
-| On the *Sections and Questions* form, once the question type is chosen and the answer added,
-  under the tab *Options*, enable *Mandatory Answer*, and set an *Error message* to be shown
-  when the user tries to submit the survey without answering the respective question.
-| Depending on the *Question Type*, the tab *Options* aggregates extra and different
-  possibilities. Examples:
-
-- *Single Line Text Box* - can choose a minimum and maximum text length (number of characters -
-  spaces do not count), and its error message.
-- *Multiple choice: multiple answers allowed*: choose between radio buttons or a dropdown menu
-  list, and if you would like to have a comment field. If the user select an answer and type a
-  comment, the values are separately recorded. Or, enable the comment field to be
-  displayed as an answer choice, with which a text type field is displayed.
-- *Matrix Type*: choose if you would like to have one or multiple choices per row.
-
-.. image:: create/sections_questions.png
+   :alt: Various tabs that can be found on the survey template page.
+
+Questions tab
+=============
+
+Add questions and sections to the survey in the :guilabel:`Questions` tab. A section will divide
+the survey into parts; to make one, simply click :guilabel:`Add a section` and type in a section
+name. Then, add questions or drag and drop questions into the divided sections.
+
+Clicking :guilabel:`Add a question` opens the :guilabel:`Create Sections and Questions` pop-up to
+create and customize the survey question.
+
+.. image:: create/survey-question-pop-up.png
```

Comment:
The width of this image needs to be cropped a little more. It's currently at a width of 975px, we need it to be 768px or less.